src/http/api/profile.ts

```diff
@@ -273,6 +273,19 @@ export function createGetProfilePostsHandler(accountService: AccountService) {
                     });
                 }
 
+                if (typeof activity.object.attributedTo === 'string') {
+                    const attributedTo = await lookupObject(
+                        apCtx,
+                        activity.object.attributedTo,
+                    );
+                    if (isActor(attributedTo)) {
```

Comment:
What if this returns falsy? We will still have the same issue right? We could just adding the post to `result.posts` in this case maybe?

Reply:
I agree. It's worth not adding it to `result.posts` in that case